### DISCUSSION TITLE
New function for generating list of all connected clients

### DISCUSSION THREAD

**Discussion by Mathiaszero** [godotengine/godot-proposals#6840] [Scripting]
In ENetMultiplayerPeer, there does not seem to be a way to generate a list of all connected clients. There should probably be something like list_connections(max size) that would generate an array of all connected ids, with specifying an optional max value in case the server has a long list of connected clients.  Because it is an array, the coder can count the array elements to get the total number also.

**Comment by AThousandShips** [comment]
I believe you can use `get_peers` on `MultiplayerAPI` and then use `get_peer` or `ENetMultiplayerPeer`
